Makefile

```diff
@@ -52,7 +52,7 @@ tag:
 	git tag -a -m "Tag as $(TAG)" -f $(TAG)
 	@echo "Tagged as $(TAG)"
 
-release: tag archive
+release: bumpver commit tag archive
```

Comment:
This should run sequentially, right? If so, it should be in the target as ``$(MAKE) bumpver`` etc. Otherwise it may run in parallel if *make* is run with e.g. ``-j4`` and fail.

Reply:
Good point, thanks! Fixed. :)
BTW, I think we might already have the same issue in other make files and should fix it here as well.